src/languageServer/javaServerStarter.ts

```
@@ -37,9 +39,14 @@ function prepareParams(): string[] {
   }
   parseVMargs(params, vmargs);
   const serverHome: string = path.resolve(__dirname, '../server');
-  const launchersFound: Array<string> = glob.sync(`**/${SERVER_NAME}`, { cwd: serverHome });
-  if (launchersFound.length) {
-    params.push('-jar'); params.push(path.resolve(serverHome, launchersFound[0]));
+  const microprofileServerFound: Array<string> = glob.sync(`**/${MICROPROFILE_SERVER_NAME}`, { cwd: serverHome });
+  const quarkusServerExtFound: Array<string> = glob.sync(`**/${QUARKUS_SERVER_EXTENSION_GLOB}`, { cwd: serverHome });
+  if (microprofileServerFound.length && quarkusServerExtFound.length) {
+    params.push('-cp');
+    params.push(`${serverHome}/*`);
+    params.push(MICROPROFILE_SERVER_MAIN_CLASS);
+  } else {
+    console.error('Unable to find required Language Server JARs');
```

we should throw an exception here, because that's a rather serious issue ;-)